CHANGELOG.md

```diff
@@ -8,6 +8,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 The intended audience of this file is for py42 consumers -- as such, changes that don't affect
 how a consumer would use the library (e.g. adding unit tests, updating documentation, etc) are not captured here.
 
+## Unreleased
+
+### Added
+
+- Functions for managing role assignment:
```

Comment:
Is there a benefit to doing `sdk.users.roles.get()`, `sdk.users.roles.add()` etc instead? Just making sure you considered it is all.

Reply:
or event `sdk.roles`

Reply:
conceptually, at least in the UI, user are added to roles and not the other way around, which is why I felt these functions should be associated with users. I think we also discussed this as a team during a grooming meeting.

I can see the argument for `users.roles`, the main reason I didn't go this way is simply because I didn't feel that roles would ever have enough functions to warrant having its own "bucket". @timabrmsn do you have thoughts here?

Reply:
I know there are planned changes to roles/permissions coming (PRM-2940), though I haven't looked through them closely enough to know how impactful that would be to this design.

I think for what we have currently, it makes sense to keep it directly under users.